Sources/MapboxSearch/InternalAPI/Engine/ApiType+Core.swift

```diff
@@ -0,0 +1,14 @@
+// Copyright © 2024 Mapbox. All rights reserved.
+
+import Foundation
+
+extension ApiType {
+    func toCore() -> CoreSearchEngine.ApiType {
+        switch self {
+        case .geocoding:
+            return .geocoding
+        case .SBS:
+            return .SBS
```

Comment:
what is the significance of the "." in ".SBS"?

Reply:
Just Swift syntax for referencing an enum case, and the type of the enum is inferred from the return value of the function's signature.